package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "aws-sqs-redriver",
+  "version": "1.0.0",
+  "description": "Re-drives messages from one SQS queue to another, e.g. to re-process \"dead letters\"",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint --cache .",
+    "build": "rm -rf lib && babel src --out-dir lib",
+    "prepublish": "npm run build",
+    "test": "NODE_ENV=test istanbul cover _mocha",
+    "test-no-cov": "NODE_ENV=test mocha",
+    "test-acceptance": "NODE_ENV=test istanbul cover _mocha -- --fgrep '/acceptance/'",
+    "test-component": "NODE_ENV=test istanbul cover _mocha -- --fgrep '/component/'",
+    "test-unit": "NODE_ENV=test istanbul cover _mocha -- --fgrep '/unit/'"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Springworks/node-aws-sqs-redriver.git"
+  },
+  "keywords": [
+    "aws",
+    "sqs",
+    "redrive",
+    "lambda"
+  ],
+  "author": "Springworks <hello@springworks.se> (http://www.springworks.se)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Springworks/node-aws-sqs-redriver/issues"
+  },
+  "homepage": "https://github.com/Springworks/node-aws-sqs-redriver#readme",
+  "devDependencies": {
+    "@springworks/test-harness": "1.4.3",
+    "babel-cli": "6.14.0",
+    "babel-core": "6.14.0",
```

Comment:
- [ ] Remove `babel-core`. It is a dependency of `babel-cli`, you don't need to depend on it directly